sdk/python/tests/api/test_integration.py

```diff
@@ -14,7 +14,11 @@
 async def test_container():
     async with dagger.Connection() as client:
         alpine = client.container().from_("alpine:3.16.2")
+<<<<<<< HEAD
```

Comment:
Missed this, (but I'm sure rebasing this PR must be a total nightmare, very understandable...)